kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobActionDropdown.js

```diff
@@ -182,9 +182,7 @@ define([
         }
 
         function updateState() {
-            const jobCountsByStatus = Jobs.getCurrentJobCounts(
-                jobManager.model.getItem('exec.jobs.byId')
-            );
+            const jobCountsByStatus = Jobs.getCurrentJobCounts(jobManager.getIndexedJobs());
```

Comment:
throughout this PR, I have switched from using `jobManager.model.getItem('exec.jobs.byId')` to using `jobManager.getIndexedJobs()`